### PR TITLE
Issue/3775 array changed behaviour

### PR DESCRIPTION
**Pull request by siemhesda**
Fix for `package.json#/config/*` array changing behavior to joining of items with double new-line

## References
Related to #3775

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -233,7 +233,8 @@
     "resetdeps": "node scripts/resetdeps.js",
     "rp-pull-request": "node scripts/update-authors.js",
     "postlint": "template-oss-check",
-    "template-oss-apply": "template-oss-apply --force"
+    "template-oss-apply": "template-oss-apply --force",
+    "start":"node index.js"
```
Comment (wraithgar) [inline package.json:237]: Where did this come from?

File: package.json [hunk 2]
```diff
@@ -264,5 +265,12 @@
   "license": "Artistic-2.0",
   "engines": {
     "node": "^18.17.0 || >=20.5.0"
+  },
+  "config":{
```
Comment (wraithgar) [inline package.json:269]: Where did this come from?

**Comment by wraithgar** [comment]
I'm confused as to what this PR is. It appears to be debugging attempts and not any code fixes.

File: index.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
-if (require.main === module) {
-  require('./lib/cli.js')(process)
-} else {
-  throw new Error('The programmatic API was removed in npm v8.0.0')
-}
+
+const { deepStrictEqual } = require("assert");
```
Comment (wraithgar) [inline index.js:2]: Changing this file breaks npm.

**Comment by siemhesda** [comment]
The changes were not supposed to go here

**Pull request closed without merging**